### PR TITLE
setup and mobile version skeleton

### PR DESCRIPTION
**Pull request by ksigei**
### In this milestone, I was able to:
- Use flexbox to layout elemets
- Use the [HTML&CSS linter]
- I was therefore able to build the following

          - the toolbar (or header):
          - the headline section (right after the header):

### CODE REVIEW

**Review by mahbubul14 — COMMENTED** [review]
# Invalid Code Review Request

**Hi @ksigei ,** 

While you made a great effort in this project, unfortunately, I cannot proceed to review your code.

You have submitted a project, that has linters error. Please fix the linters error and submit your project again.

Your Code Review Request will be marked as invalid in your Dashboard, so please submit a new one once you are ready 🙏

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

--------

_Invalid Code Review Request **does not count** into the code reviews limit._
```

File: index.html [hunk 1]
```diff
@@ -0,0 +1,34 @@
...
+    <link rel="stylesheet" href="style.css">
+</head>
+<body>
+    <header class="header-container">
+        <div class="header-img-left">
+            <img src="images/image_geometry_1.png" alt="image">
+        </div>
+        <nav class="header-nav-lg">
+            <img src="images/ic_menu.jpg" alt="image">
+        </nav>>
```
Comment (mahbubul14) [inline index.html:17]: - [ ] I couldn't see your hamburger button on the top right of your page. Please fix it. :writing_hand:

File: index.html [hunk 2]
```diff
@@ -0,0 +1,34 @@
...
+        <div class="header-my-name">
+            <h1>Kiprono Japhet</h1>
+        </div>
+        <div class="header-about">
+            <p class="aboutme">Hello! I am a software developer! I can help you build a product, feature or website. Taka a look of my works.
+                  If you like what you see and have a project you need coded, don’t hesitate and contact me.
+            </p>
+        </div>
+        
+        <button>Start collaboration</button>
```
Comment (mahbubul14) [inline index.html:28]: - [ ] Please Implement the button interactions (enable, hover, etc.) for this button. :writing_hand:

**Review by mahbubul14 — CHANGES_REQUESTED** [review]
# Required Changes ♻️

Hi @ksigei  , 

Good job so far! :+1: 
There are some issues that you still need to work on to go to the next project but you are almost there!

- [ ]  Kindly use Flexbox to layout the elements on the page. :computer: 

- [ ] Please use [CSS Media Queries](https://www.w3schools.com/css/css3_mediaqueries.asp) for your mobile version. It is not currently optimized for the mobile version.

- [ ] Check the comments under the review for more suggestions. :arrow_down: 

###  To Highlights
:heavy_check_mark: No linters error.
:heavy_check_mark: Github flow used properly.

### Optional suggestions

- [ ] Please use descriptive commit messages - each commit should be small enough to describe it in one sentence. The commit message should be written in the imperative tense and capitalized. [Read more about how to write descriptive commit messages.](https://www.freecodecamp.org/news/level-you-up-to-awesome-commit-messages-a85558cb90e8)

- [ ] For readability, please add two spaces of indentation. Do not use the tab key. You did it throughout your code in the `HTML` file. Please fix it. For the HTML & CSS best practices we should follow theis rules. :writing_hand: 

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification. You can also reach me on [slack](https://microverse-students.slack.com/team/U01J39GL614)

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._
```

File: style.css [hunk 3]
```diff
@@ -0,0 +1,91 @@
...
+  }
+
+  .header-nav-lg a {
+    color: #c4c4c4;
+  }
+
+  .about-container {
+    display: flex;
+    flex-direction: column;
+    width: 288px;
```
Comment (Wes-Isaac) [inline style.css:26]: - [ ] _[OPTIONAL]_ Setting a fixed width is creating an unnecessary space in your site please fix that.

**Review by Wes-Isaac — APPROVED** [review]
Hi @ksigei,

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

You can also consider:
_Check the comments under the review._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**